backend/src/main/java/gov/cdc/usds/simplereport/api/testresult/TestResultDataResolver.java

```diff
@@ -28,7 +28,7 @@ public Person getPatient(TestEvent testEvent) {
   }
 
   public Date getDateAdded(TestEvent testEvent) {
-    return testEvent.getTestOrder().getCreatedAt();
+    return testEvent.getDateTested();
```

Comment:
That does seem to make more sense, yes. I suspect we just didn't update this when we should have.